### PR TITLE
Fix Quick Start bottom padding on the My Site screen

### LINKED ISSUE #15430: Quick Start adds bottom padding to My Site screen
Parent #15296

### Expected behavior

The Floating Action Button (FAB) should always float above content on the My Site screen.

### Actual behavior

An unwanted bottom margin appears on the My Site screen after tapping the “Visit your site” step in the Quick Start:

Screen recording gif showing unwanted bottom margin 

### Steps to reproduce the behavior

1. Follow the Quick Start steps after creating a new site
2. Tap the “Visit your site” step in the Quick Start
3. Notice that the FAB is no longer floating above screen content, but instead over a white padding area which shouldn't be there.

### PR DESCRIPTION
**Pull request by ashiagr**
Fixes #15430

https://user-images.githubusercontent.com/1405144/143029388-c3699106-7989-4895-b95b-5d0c80b09c63.mp4

Cc @osullivanchris 

To test:

- Follow the Quick Start steps after creating a new site.
- Tap the “Visit your site” step in the Quick Start so that a snack bar gets displayed.
- Wait for the snackbar to auto-dismiss.
- Notice that the blank area occupied by the snackbar (that was shown earlier even after snackbar dismiss) is now not shown (see timestamp `0.06` in the video).

Review Instructions:

Only 1 reviewer is sufficient.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by ParaskP7** [comment]
Update: There has been discussion about the overall solution in the second #15614 PR, which actually fixes the problem in this PR, but by introducing another problem, see comments [here](https://github.com/wordpress-mobile/WordPress-Android/pull/15614#pullrequestreview-814680400) and [here](https://github.com/wordpress-mobile/WordPress-Android/pull/15614#issuecomment-977734203).

Will wait on any updates before proceeding with another code review.

**Comment by ashiagr** [comment]
👋 @ParaskP7, @osullivanchris  

Both issues #15430, #15461 are fixed in this PR. 🎉 Additionally, reduced `My Site` recyclerview's bottom padding.  If everything looks good in testing, we can merge this PR and close #15614 PR without a merge.

`Timestamp 0.04` - Fixed [Dead pixels around the snack bar issue](https://github.com/wordpress-mobile/WordPress-Android/issues/15461) 952c183
`Timestamp 0.06` - Fixed [Quick Start adds bottom padding to My Site screen](https://github.com/wordpress-mobile/WordPress-Android/issues/15430) 9e0cff6
`Timestamp 0.09` - Reduced `My Site` recycler view's bottom padding c0237db

https://user-images.githubusercontent.com/1405144/143540928-1a524f35-51f4-40bf-9900-5e6ba7077f52.mp4

File: WordPress/src/main/res/layout/main_activity.xml
Comment (ParaskP7) [inline WordPress/src/main/res/layout/main_activity.xml]: Minor (🔍): This comment ended up being on the same line as the `<?xml ... ?>` definition due to the automatic refactoring. Consider moving it back to the below line.

File: WordPress/src/main/res/layout/main_activity.xml
Comment (ParaskP7) [inline WordPress/src/main/res/layout/main_activity.xml:18]: Praise (❤️): What a nice find! 🤯

File: WordPress/src/main/res/values/dimens.xml
Comment (ParaskP7) [inline WordPress/src/main/res/values/dimens.xml:311]: Praise (❤️): Another great win! 🥇

**Review by ParaskP7 — APPROVED** [review]
👋 @ashiagr !

I have reviewed and tested this PR as per the instructions, everything works as expected, amazing job, all 3 of our problems are now gone, you rock! 🌟 🌟 🌟

-----

I have left only one minor (🔍) comments for you to consider. I am going to approve this PR anyway, since none is blocking. I am **NOT** going to merge this PR yet to give you some time to apply any of my suggestions. However, feel free to ignore them and merge the PR yourself.

-----

PS: I am not sure if you want @osullivanchris to quickly review this before merging, so please feel free to merge it yourself when you are ready, 🟢 light from my side! 🥇

File: WordPress/src/main/res/layout/main_activity.xml
Comment (ashiagr) [reply WordPress/src/main/res/layout/main_activity.xml]: Ohh yesss! Moved back in 43d927d.

Comment (ParaskP7) [reply WordPress/src/main/res/layout/main_activity.xml]: Awesome! 🙏

**Review by osullivanchris — APPROVED** [review]
Downloaded the apk. I can no longer see the dead pixels around the snackbar, and similarly when the snackbar goes away, the UI looks correct (no blank space).

Great to see this fixed 🎉

**Comment by ashiagr** [comment]
Thank you both! @osullivanchris , @ParaskP7 

Merging it now.

**Pull request merged**
